lib/action_dispatch/session/active_record_store.rb

```diff
@@ -148,6 +148,7 @@ def logger
   end
 end
 
+require 'active_record'
```

Comment:
Can you move this to the `active_record/session_store.rb` file?

Reply:
Done 👍